File: app/src/main/java/com/example/vendo/GlobalVariables.java
package com.example.vendo;

import android.app.Application;

public class GlobalVariables extends Application {
    private String languageCode;

    public String getLanguageCode() {
        return languageCode;
    }

    public void setLanguageCode(String someVariable) {
        this.languageCode = someVariable;
    }
}

File: app/src/main/java/com/example/vendo/GlobalVariables.kt
package com.example.vendo

import android.app.Application

class GlobalVariables : Application() {
    var languageCode: String? = "en"
    fun findLanguage(){
        languageCode = this.getSharedPreferences(getString(R.string.app_name), MODE_PRIVATE)
            .getString("languageCode","")
    }
}